waiter/src/waiter/settings.clj

```diff
@@ -70,6 +70,9 @@
                                                   (s/required-key :async-request-max-status-checks) schema/positive-int
                                                   (s/required-key :async-request-max-timeout-ms) schema/positive-int
                                                   (s/required-key :async-request-timeout-ms) schema/positive-int
+                                                  (s/required-key :byte-buffer-capacity-factor) (s/maybe schema/non-negative-int)
+                                                  (s/required-key :byte-buffer-heap-percent) (s/maybe schema/positive-num)
```

Comment:
Should this be limited to (0,100]? Or some other hard max less than 100?

Reply:
sgtm, will change. Actually need to be clearer that this is in the range (0, 1]

Reply:
Changed to enforce byte-buffer-heap-percent is a positive fraction less than or equal to 1